astronomer/providers/google/cloud/triggers/bigquery.py

```diff
@@ -378,3 +382,83 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                 self.log.exception("Exception occurred while checking for query completion")
                 yield TriggerEvent({"status": "error", "message": str(e)})
                 return
+
+
+class BigQueryTableExistenceTrigger(BaseTrigger):
+    """Initialise the BigQuery Table Existence Trigger with needed parameters"""
+
+    def __init__(
+        self,
+        project_id: str,
+        dataset_id: str,
+        table_id: str,
+        google_cloud_conn_id: str,
+        hook_params: Dict[str, Any],
+        poll_interval: float = 4.0,
+    ):
+        super().__init__()
```

Comment:
`super().__init__()` can be removed as BaseTrigger is ABC.

Reply:
It's no harm keeping it. Removed it